### ISSUE TITLE
declaration-block-semicolon-space-before ignores semicolons after less mixins

### ISSUE THREAD

**Issue by kachkaev** [stylelint/stylelint#1607]
It seems like there is a problem with handling `declaration-block-semicolon-space-before` around mixins, even in 6.9.0.

## MWE

`test.less`

```less
.class1 {
  color: green;
  .mixin();
  .mixin2;
}

.class2 {
  color: green ;
  .mixin() ;
  .mixin2 ;
}
```

### Rule: never
`.stylelintrc`
```json
{
  "rules": {
    "declaration-block-semicolon-space-before": "never" 
  }
}
```

```bash
$ node_modules/.bin/stylelint --version
6.9.0
$ node_modules/.bin/stylelint test.less --syntax less

test.less
 8:15  ✖  Unexpected whitespace before ";"   declaration-block-semicolon-space-before

```
__Expected output__
```
test.less
 8:15  ✖  Unexpected whitespace before ";"   declaration-block-semicolon-space-before
 9:11  ✖  Unexpected whitespace before ";"   declaration-block-semicolon-space-before
10:10  ✖  Unexpected whitespace before ";"   declaration-block-semicolon-space-before
```

### Rule: always
`.stylelintrc`
```json
{
  "rules": {
    "declaration-block-semicolon-space-before": "always" 
  }
}
```

```bash
$ node_modules/.bin/stylelint test.less --syntax less

test.less
 2:14  ✖  Expected single space before ";"   declaration-block-semicolon-space-before

```
__Expected output__
```
test.less
 2:14  ✖  Expected single space before ";"   declaration-block-semicolon-space-before
 3:10  ✖  Expected single space before ";"   declaration-block-semicolon-space-before
 4:9   ✖  Expected single space before ";"   declaration-block-semicolon-space-before
```

**Comment by jeddy3** [comment]
The two previous Less semicolon issues (#1409 and #1389) were closed in #1601 by a change in `postless-parser` that meant semicolons after Less syntax were *simply ignored*. This is arguably in-keeping with how we're dealt with non-standard syntax elsewhere.

That is also what is happening here. So, we can either close this issue (with a [docs update](https://github.com/stylelint/stylelint/pull/1601/files#diff-0800f37bb02af849a8cdb0fb8be93590L10)) because the behaviour here is consistent with the previous fix or we [change this test](https://github.com/stylelint/stylelint/blob/master/src/rules/no-extra-semicolons/__tests__/index.js#L487-L489) and create an issue for `no-extra-semicolons` too.

**Comment by evilebottnawi** [comment]
@jeddy3 duplicate https://github.com/stylelint/stylelint/issues/1608, because parser not work correctly, we should close this issue, rename https://github.com/stylelint/stylelint/issues/1608 to more correctly and create issue in `postcss-less`
